Cargo.toml

```diff
@@ -46,23 +49,21 @@ version = "0.1.3"
 
 [dev-dependencies.crc16]
 default-features = false
-version = "0.3.4"
+version = "0.4.0"
 
 [dev-dependencies.either]
 default-features = false
-version = "1.4.0"
+version = "1.5.0"
 
 [dev-dependencies.jnet]
-git = "https://github.com/japaric/jnet"
-rev = "df96b408049ca952ad7844d6552e87cf8fc18d2a"
+#git = "https://github.com/japaric/jnet"
+git = "https://github.com/burrbull/jnet"
```

Comment:
Not happy about this and the line above. Can we crate jnet?

Reply:
The examples using this crate does not really have any reason to be in this crate. What about removing all the examples that rely on not published on crates.io crates?

Reply:
@therealprof OK